lms/djangoapps/course_home_api/progress/v1/tests/test_views.py

```diff
@@ -35,6 +35,8 @@ def test_get_authenticated_enrolled_user(self, enrollment_mode):
         self.assertIsNotNone(response.data['courseware_summary'])
         for chapter in response.data['courseware_summary']:
             self.assertIsNotNone(chapter)
+        self.assertIn(str(self.course.id), response.data['studio_url'])
+        self.assertIn('settings/grading', response.data['studio_url'])
```

Comment:
I think you might be able to combine these to something like `str(self.course.id) + '/settings/grading'` or something like that. If you print out the studio_url, you can see what it should look like (i.e. it might be `'/settings/grading/' + str(self.course.id)`